### ISSUE TITLE
The project can not be configured in the studio，could you help me ?

### ISSUE THREAD

**Issue by ZousTeam** [nickbutcher/plaid#38]
I import the progrect into studio, but some config is wrong ,it cann`t be compiled normally, so could you  help me ?

**Comment by nickbutcher** [comment]
Can you be more specific with what is wrong?  Is this different to #17? Do you have git on your path?

**Comment by ZousTeam** [comment]
![h_ w ec k gt7h yw0 q](https://cloud.githubusercontent.com/assets/15122546/11139416/0221958a-8a0a-11e5-8b86-0e8314e6bd76.png)
![tg upio h t0rif eck et](https://cloud.githubusercontent.com/assets/15122546/11139417/0249d63a-8a0a-11e5-90af-c6c2ac559db6.png)
it is the problem

**Comment by ZousTeam** [comment]
Thank you for your help, i have resolved this problem, the git path has not been put into system environment under window

**Comment by BillBosiolis** [comment]
On Ubuntu, with Git installed (and being on the path) I had no problem to open the project in Android Studio and build it successfully.

On Windows, I was getting the known error (20 - CreateProcess error=2). Here's what I did to solve it

1. Installed a newer Git version (I don't think it has to do anything with the version, I just wanted to modify the installation)
2. In the "Adjusting your Path environment" step I selected the second option - Run Git from the Windows Command Prompt
3. Restarted Studio and set the the path to the git executable. Settings -> Version Control -> Git and set it "C:\Program Files\Git\cmd\git.exe".

Hope that it helps
